liblangutil/Exceptions.h

```diff
@@ -256,6 +256,11 @@ class Error: virtual public util::Exception
 		return false;
 	}
 
+	static bool hasErrorsWarningsOrInfos(ErrorList const& _list)
```

Comment:
Perhaps we should also such a helper in `ErrorReporter` as well? It would be analogous to `errorReporter.hasErrors()` that we have there already.

Reply:
Done. I've changed the occurrences of this helper with the newly created one.